drupal-check

```diff
@@ -10,10 +10,12 @@ use Composer\XdebugHandler\XdebugHandler;
 
 gc_disable();
 
-$composerAutoloadFile = __DIR__ . '/vendor/autoload.php';
```

Comment:
I don't have integration tests running. I feel like this magical global is required. Or did this work for you?